### PR TITLE
LPS-173139 Long web content titles are not readable in Widget and Content pages

### PR DESCRIPTION
**Pull request by BarbaraCabrera**
[Link to ticket](https://issues.liferay.com/browse/LPS-173139)

## Test Scenario 1

* Navigate to Content & Data > Web Content and create two of them with the names:
	- Company_Liferay_article_new_week_Monday_1 	
	- Company_Liferay_article_new_week_Monday_2
* Navigate to Site Builder > Pages and create one Widget page.
* Navigate to the page and click on the add button.
* Go to “Content” tab and check te tooltip for both and the tooltip of the add button on the right side.

Test WP
Liferay DXP

## Test Scenario 2

* Navigate to Content & Data > Web Content and create one with the name:
	- Company_Liferay_article_new_week_Monday_1
* Navigate to Site Builder > Pages and create one Content page.
* Add one content display and select the web content.
* Check the tooltip with the name in the page content tab.

image

### CODE REVIEW

**Comment by liferay-echo** [comment]
Thank you for submitting this PR. 👏

Make sure it follows the [Echo Pull Request Template](https://gist.github.com/liferay-echo/5a4455285ff1800f9da146816d85e67d#file-pull-request-template-md) if you haven't already done so. 

Learn more about it [here](https://liferay.atlassian.net/wiki/spaces/ENGECHO/pages/2027455014/Echo+Pull+Request+Etiquette).

**Comment by BarbaraCabrera** [comment]
ci:test:sf

**Comment by BarbaraCabrera** [comment]
ci:test:sf

**Comment by BarbaraCabrera** [comment]
ci:test:sf

**Comment by BarbaraCabrera** [comment]
ci:test:sf

**Comment by BarbaraCabrera** [comment]
ci:test:sf

**Comment by BarbaraCabrera** [comment]
ci:test:sf

File: modules/apps/product-navigation/product-navigation-control-menu-web/src/main/resources/META-INF/resources/js/TabItem.js [hunk 1]
```diff
@@ -62,7 +62,10 @@ const TabItem = ({item}) => {
 			})}
 			ref={item.disabled ? null : sourceRef}
 		>
-			<div className="sidebar-body__add-panel__tab-item-body">
+			<div
+				className="lfr-portal-tooltip sidebar-body__add-panel__tab-item-body"
```
Comment (sandrodw3) [inline modules/apps/product-navigation/product-navigation-control-menu-web/src/main/resources/META-INF/resources/js/TabItem.js:66]: I think it's not needed here neither 😛

File: modules/apps/product-navigation/product-navigation-control-menu-web/src/main/resources/META-INF/resources/js/TabItem.js [hunk 2]
```diff
@@ -81,10 +84,11 @@ const TabItem = ({item}) => {
 			{!item.disabled && (
 				<ClayButton
```
Comment (sandrodw3) [inline modules/apps/product-navigation/product-navigation-control-menu-web/src/main/resources/META-INF/resources/js/TabItem.js:85]: Could we take advantage and add `aria-label` with the long `title` for the screen readers? I think it's more correct that the `<span>` with `sr-only` (we can remove it if we add the `aria-label`) 😄

**Review by sandrodw3 — CHANGES_REQUESTED** [review]
This is cool! I just suggested some small improvements 🥳

I also noticed some things I would like to comment (probably we could address them in another subtask and PR):

- In my case, in widget pages, the title is being truncated, maybe we should investigate it 🤔
image
- I noticed that the markup of the list items in both kind of pages is a bit weird with long names. Plus button is too close to the name and the ellipsis button is like outside the hovered background. Maybe we could take advantage and improve the markup with some flex classes 😛
image
image

**Comment by BarbaraCabrera** [comment]
ci:test:sf

**Comment by sandrodw3** [comment]
> > This is cool! I just suggested some small improvements partying_face
> > I also noticed some things I would like to comment (probably we could address them in another subtask and PR):
> > 
> > * In my case, in widget pages, the title is being truncated, maybe we should investigate it thinking
> >   image
> > * I noticed that the markup of the list items in both kind of pages is a bit weird with long names. Plus button is too close to the name and the ellipsis button is like outside the hovered background. Maybe we could take advantage and improve the markup with some flex classes stuck_out_tongue
> >   image
> >   image
> 
> Hi Sandro! For the WP case, I've seen that the problem comes for a shorten added in AddContentPanelDisplayContext.java (L.196), i'll send a fix for that 👍🏼
> 
> And about the markup of the list items, I'll upload a commit to improve them, thanks! smile

Super cool, thanks a lot! 😍

**Comment by BarbaraCabrera** [comment]
ci:test:sf

**Review by sandrodw3 — APPROVED** [review]

**Comment by sandrodw3** [comment]
ci:forward

**Comment by sandrodw3** [comment]
ci:forward

**Comment by Tim-Cao** [comment]
Hi @BarbaraCabrera, would you mind checking the unit test failure?
[apps:layout:layout-content-page-editor-web:packageRunTest](https://test-1-37.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/482460//consoleText)
Thanks

**Comment by sandrodw3** [comment]
ci:forward

**Pull request closed without merging**